$OpenBSD: patch-src_init-cl_lisp,v 1.1 2013/01/04 14:05:41 edd Exp $

Fix a bug about CL-USER:*MAXIMA-BUILD-TIME* being undefined.
Patch comes from upstream git.

--- src/init-cl.lisp.orig	Wed Dec 12 10:46:33 2012
+++ src/init-cl.lisp	Wed Dec 12 10:46:39 2012
@@ -5,11 +5,16 @@
 ;; author:      Liam Healy <Liam.Healy@nrl.navy.mil>
 ;;********************************************************
 
-(in-package :maxima)
-
 ;;; An ANSI-CL portable initializer to replace init_max1.lisp
 
-#+ecl (defvar cl-user::*maxima-build-time* '#.(multiple-value-list (get-decoded-time)))
+;; CL-USER:*MAXIMA-BUILD-TIME* is defined in maxima.asd and maxima.system,
+;; but I guess ECL doesn't see that, so define it here.
+#+ecl (progn
+  (in-package :cl-user)
+  (defvar *maxima-build-time* '#.(multiple-value-list (get-decoded-time)))
+  (export '*maxima-build-time*))
+
+(in-package :maxima)
 
 ;;; Locations of various types of files. These variables are discussed
 ;;; in more detail in the file doc/implementation/dir_vars.txt. Since
